src/main/resources/schema.sql

```diff
@@ -1,6 +1,24 @@
+CREATE TABLE IF NOT EXISTS people (
+    id BIGINT generated always as identity PRIMARY KEY,
+    first_name TEXT not null,
+    last_name TEXT not null
+);
+
 CREATE TABLE IF NOT EXISTS sales (
-    id SERIAL PRIMARY KEY,
-    price NUMERIC(10,2) NOT NULL,
-    percentage FLOAT default 10.00,
-    commission NUMERIC(10,2) not null
-)
\ No newline at end of file
+    id BIGINT generated always as identity PRIMARY KEY,
+    price NUMERIC(10,2) not null,
+    date_stamp timestamp with TIME ZONE DEFAULT NOW(),
```

Comment:
If you want (personal preference) you can abbreviate the timestamp with TIME ZONE as `TIMESTAMPTZ`

Reply:
Cool. I ll do that.  I have read the differences between the two too.